code/modules/projectiles/projectile/bullets.dm

```diff
@@ -272,15 +266,30 @@
 	explosion(target, -1, 0, 1)
 	return TRUE
 
+/obj/item/projectile/bullet/pellet
+	var/tile_dropoff = 0.75
+	var/tile_dropoff_s = 1.25
+
 /obj/item/projectile/bullet/pellet/shotgun_buckshot
 	name = "buckshot pellet"
 	damage = 12.5
 
 /obj/item/projectile/bullet/pellet/shotgun_rubbershot
+	name = "rubbershot pellet"
 	damage = 3
 	stamina = 25
 
+/obj/item/projectile/bullet/pellet/Range()

```

Comment:
This was changed to specifically shotguns i believe

Reply:
? I'm moving this from the top of the file down because @LanCartwright put it at the top for some reason, it's been modified to take in account rubbershot because it was bugged.